Sources/Classes/core/JavascriptBridge/MiniAppScriptMessageHandler.swift

```diff
@@ -200,6 +205,36 @@ internal class MiniAppScriptMessageHandler: NSObject, WKScriptMessageHandler {
         }
     }
 
+    func getContactId(messageId: String) {
```

Comment:
Can we rename it to `callbackId` ? Would be more precise

Reply:
sure makes sense, updated it